recipes/rustybam/meta.yaml

```diff
@@ -25,12 +23,19 @@ requirements:
   host:
     - gsl
     - libcblas
-    - openssl
-    - zlib
+    - libcurl
     - bzip2
-    - htslib
+    - xz
+    - zlib
+    - libdeflate
+    - openssl  # [not osx]
```

Comment:
why is that needed for OSX?

Reply:
I couple weeks ago I was trying to get MacOS to work and I was getting build errors. Then I thought to check the `htslib` recipe since `rustybam` uses `htslib`.  And that is where I found that bit of code and assumed that `htslib` itself would be configured correctly?

https://github.com/bioconda/bioconda-recipes/blob/master/recipes/htslib/meta.yaml